android/quest/src/main/java/org/smartregister/fhircore/quest/configuration/view/PatientDetailsViewConfiguration.kt

```diff
@@ -27,20 +27,21 @@ data class PatientDetailsViewConfiguration(
   override val classification: String,
   val contentTitle: String = "Responses",
   val valuePrefix: String = "G6PD ",
```

Comment:
why is g6pd hard coded here?

Reply:
This is just a hint, we can change to something generic. We are loading the actual configurations from [patient_details_view_configuration.json](https://github.com/opensrp/fhircore/blob/19b4889305146d07016a69721416677fa800f404/android/quest/src/main/assets/configurations/view/patient_details_view_configurations.json#L11) file

Reply:
ah got it cool, I always forget about the hints!

yea might as well change to something that explains how it should be used